### ISSUE TITLE
4-Tracks

### ISSUE THREAD

**Issue by MariaTerzieva** [HackBulgaria/Ruby-on-Rails-1#2]
Can you give an example of what the `jazz_playlist` and what the `house_playlist` contain and what `jazz_playlist & house_playlist` contains and also what `jazz_playlist - house_playlist` contains?

Also is "I want you to test out an idea. Implement Playlist#each, that returns an enumerator if no block given. That way, we can get all the benefits of including Enumerable, without polluting the class with a ton of methods." just recommendable or is it a requirement?

**Comment by gsamokovarov** [comment]
`Track#genre` should be a plain string, so there is nothing fancy going on with the genres. The example isn't really good enough, so sorry for that.

`jazz_playlist & house_playlist` should return all the tracks that are both in `jazz_playlist` and in `house_playlist`.

`jazz_playlist - house_playlist` should return a playlist with all the tracks that are in `jazz_playlist`, but not in the `house_playlist`.

Basically, those are the usual set theory operations.

Thanks for the feedback. I cleaned up the problem definition a bit. Hopefully, its a bit more understandable. If you have any more question, don't hesitate to ask.

**Comment by gsamokovarov** [comment]
@MariaTerzieva did you find your answer? Can I close the issue?

**Comment by MariaTerzieva** [comment]
@gsamokovarov, yes. Thank you! You can close it.
